src/Validator/UniqueDonationSubscriptionValidator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace AppBundle\Validator;
+
+use AppBundle\Donation\DonationRequest;
+use AppBundle\Repository\DonationRepository;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+class UniqueDonationSubscriptionValidator extends ConstraintValidator
+{
+    private $donationRepository;
+    private $urlGenerator;
+
+    public function __construct(DonationRepository $donationRepository, UrlGeneratorInterface $urlGenerator)
+    {
+        $this->donationRepository = $donationRepository;
+        $this->urlGenerator = $urlGenerator;
+    }
+
+    /**
+     * @param DonationRequest $value
+     * @param UniqueEntity    $constraint
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        if (!$constraint instanceof UniqueEntity) {
+            throw new UnexpectedTypeException($constraint, UniqueCommittee::class);
```

Comment:
`UniqueCommittee`??

Reply:
Ooooo, my bad :s , I will wait @ottaviano response for the next com before correct it. But thx ;)